uaclient/event_logger.py

```diff
@@ -145,23 +166,48 @@ def _generate_failed_services(self):
         }
         return list(set.union(self._failed_services, services_with_error))
 
+    def _process_events_services(self):
+        response = {
+            "_schema_version": JSON_SCHEMA_VERSION,
+            "result": "success" if not self._error_events else "failure",
+            "processed_services": sorted(self._processed_services),
+            "failed_services": sorted(self._generate_failed_services()),
+            "errors": self._error_events,
+            "warnings": self._warning_events,
+            "needs_reboot": self._needs_reboot,
+        }
+
+        print(json.dumps(response, sort_keys=True))
+
+    def _process_events_status(self):
+        output = format_machine_readable_output(self._output_content)
+        output["result"] = "success" if not self._error_events else "failure"
+        output["errors"] = self._error_events
+        output["warnings"] = self._warning_events
+
+        if self._event_logger_mode == EventLoggerMode.JSON:
+            from uaclient.util import DatetimeAwareJSONEncoder
```

Comment:
Can't we import this on beginning of the file ?

Reply:
Unfortunately no - util has circular imports :/